src/hooks/transfers/useTransfers.tsx

```diff
@@ -28,10 +27,45 @@ import { useAppSelector } from '@/state/appTypes';
 
 import { convertBech32Address } from '@/lib/addressUtils';
 import { isNativeDenom } from '@/lib/assetUtils';
+import { MustBigNumber } from '@/lib/numbers';
 
 import { useAccounts } from '../useAccounts';
+import { useDebounce } from '../useDebounce';
 import { useSkipClient } from './skipClient';
 
+type TransferQueryFnProps = {
+  queryKey: [string, { skipClient: SkipClient }];
+};
+
+export const chainsQueryFn = async ({ queryKey }: TransferQueryFnProps) => {
+  const [, { skipClient }] = queryKey;
+  const skipSupportedChains = await skipClient.chains({
+    includeEVM: true,
+    includeSVM: true,
+  });
+  const chainsByNetworkMap = skipSupportedChains.reduce<Record<string, Chain[]>>(
```

Comment:
I don't like Record<string, ?> because it incorrectly types accesses to never return undefined which is a lie. Prefer { [key: string]: T } syntax.

Reply:
I prefer Record only when the keys are a known set of constant strings.